### ISSUE TITLE
set font

### ISSUE THREAD

**Issue by gh1232** [philc/vimium#4123]
is it possible to do these:

map 1 set font.minimum-size.x-western = 24

map 8 set font.name-list.serif.x=western = "lucinda"

i would like to change font and font size on various sites i visit

thanks

**Comment by gdh1995** [comment]
No, because web-extensions can not change Firefox's `about:config` configuration.
